ssp

```diff
@@ -2045,16 +2045,18 @@ sub print_ea3_php_configuration {
         }
 
         if ( $EA3_PHP4HANDLER && $EA3_PHP4HANDLER ne 'none' ) {
-            my @php_v = split /\n/, timed_run( 0, '/usr/local/php4/bin/php', '-v' );
-            if ( @php_v && $php_v[0] =~ /^PHP\s(\S+)\s(\S+)/ ) {
-                $EA3_PHP4VERSION = $1;
-            }
-            else {
-                $EA3_PHP4VERSION = '(version unknown)';
-            }
+            if ( -x '/usr/local/php4/bin/php' ) {
+                my @php_v = split /\n/, timed_run( 0, '/usr/local/php4/bin/php', '-v' );
+                if ( @php_v && $php_v[0] =~ /^PHP\s(\S+)\s(\S+)/ ) {
```

Comment:
More concisely written as:

`($EA3_PHP4VERSION) = $php_v[0] =~ /^PHP\s(\S+)\s(\S+)/ if @php_v;`
`$EA3_PHP4VERSION ||= '(ea3 php4 version unknown)';`

Aside from the above recreational golfing (feel free to ignore), why do we need the second capture group?  Doesn't look like we're using it.